libs/cdk/src/lib/components/forms/directives/input/input.directive.ts

```diff
@@ -274,6 +274,10 @@ export class InputDirective<
         takeUntil(this._destroy$),
       )
       .subscribe();
+
+    if (this._elementRef.nativeElement.className.includes('et-date-input')) {
```

Comment:
This really should not depend on a magic class string.
We should rather do something like Angular Material
https://github.com/angular/components/blob/382cc50ab8ec986413aef5d4ca0b9e7808271eea/src/material/input/input.ts#L267